src/attributes.jl

```diff
@@ -337,6 +337,17 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    LogLevel(value::Int)
+
+An optimizer attribute for level of verbosity of the optimizer. `0` means
+no output and for positive values, the higher the values the more output is
+allowed.
```

Comment:
Say explicitly that the meaning of the output levels is solver-specific except for `0`?